docs/semgrep-code/findings.md

```diff
@@ -67,7 +67,7 @@ Use filters to narrow down your results. The following criteria are available fo
 | **Branches**           | Filter by findings in different Git branches. |
 | **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with RBAC enabled. |
 | **Tags**               | Filter for findings based on the tags associated with the project. |
-| **Status**             | Filter the triage state of a finding. Refer to [Triage statuses](/semgrep-code/triage-remediation#triage-statuses) to understand triage states. |
+| **Status**             | Filter the triage state of a finding. Refer to [Triage statuses](/semgrep-code/triage-remediation#triage-statuses) to understand triage states. You can filter findings with a status of **Ignored** further by reason:  **False positive**, **Acceptable risk**, **No time to fix**. |
```

Comment:
Might not be necessary, but since we also show `No triage reason`, do you think we should also include that option? Or do you think that's straightforward enough to leave as is?

Reply:
Good call -- I will add this status